src/controllers/orderlistController.js

```diff
@@ -0,0 +1,26 @@
+const { orderlistService } = require('../services');
+const { throwError } = require('../utils');
+const { v4: uuidv4 } = require('uuid');
+
+const userCheckOrderController = async (req, res) => {
+  const userId = req.params.userId;
+  try {
+    const checkUserOrder = await orderlistService.getCheckOrderList(userId);
```

Comment:
```suggestion
    const checkUserOrder = await orderService.getCheckOrderList(userId);
```

order데이터를 관리하는 service에서 주문 '목록'만을 관리하지 않을 것이기 때문에 전제 파일(router, controller, service, model)에서 list라는 단어는 제거해 주세요 :)